### PR TITLE
Add Shor UI

### PR DESCRIPTION
**Pull request by lharzenetter**

### CODE REVIEW

**Comment by lharzenetter** [comment]
depends on https://github.com/OpenTOSCA/tosca-common-artifact-types/pull/6

File: nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/Qiskit-App-Implementation_w1/NodeTypeImplementation.tosca [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypeimplementations" id="nodeTypeImplementationsQuantil-Qiskit-App-Implementation_w1">
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypeimplementations" id="nodeTypeImplementationsQuantil-Qiskit-App-Implementation_w1">
```
Comment (wederbn) [inline nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/Qiskit-App-Implementation_w1/NodeTypeImplementation.tosca]: Shouldn´t this still be winery?

File: nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/Qiskit-App-Implementation_w1/NodeTypeImplementation.tosca [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
             <ImplementationArtifact xmlns:ToscaBaseTypes="http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes" xmlns:artifactTemplatesQuantil="https://ust-quantil.github.io/artifacttemplates" name="Qiskit_App_Start" interfaceName="http://opentosca.org/interfaces/lifecycle" operationName="start" artifactType="ToscaBaseTypes:ScriptArtifact" artifactRef="artifactTemplatesQuantil:Qiskit-App_Start-w1"/>
         </ImplementationArtifacts>
         <DeploymentArtifacts>
-            <DeploymentArtifact xmlns:ToscaBaseTypes="http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes" xmlns:artifactTemplatesQuantil="https://ust-quantil.github.io/artifacttemplates" name="Qiskit_App_Adapter_DA" artifactType="ToscaBaseTypes:ScriptArtifact" artifactRef="artifactTemplatesQuantil:Qiskit-App-Adapter_DA-w1"/>
+            <DeploymentArtifact xmlns:artifactTypesQuantil="https://ust-quantil.github.io/artifacttypes" xmlns:artifactTemplatesQuantil="https://ust-quantil.github.io/artifacttemplates" name="Qiskit_App_Adapter_DA&#x9;" artifactType="artifactTypesQuantil:QiskitApp" artifactRef="artifactTemplatesQuantil:Qiskit-App-Adapter_DA-w1"/>
```
Comment (wederbn) [inline nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/Qiskit-App-Implementation_w1/NodeTypeImplementation.tosca]: The name somehow changed strangely :D

**Review by wederbn — APPROVED** [review]
Two minor comments, apart from that LGTM

**Review by wederbn — APPROVED** [review]
LGTM

**Pull request merged**